pkg/seeds/seeds.go

```diff
@@ -9,17 +9,21 @@ import (
 	"gorm.io/gorm"
 )
 
-func SeedRepositoryConfigurations(db *gorm.DB, size int) error {
+func SeedRepositoryConfigurations(db *gorm.DB, size int, orgID string) error {
```

Comment:
Some food for thought to get your opinion on.  I could see in the future us wanting add more seed options, i wonder if i makes sense to have a SeedOptions struct.  That way orgId could default to "", and every person calling it wouldn't have to specify it.  We could add additional options in the future without having to update all callers.

Reply:
Sounds good to me